Kernel/API/POSIX/sys/resource.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2022, Lucas Chollet <lucas.chollet@free.fr>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <Kernel/API/POSIX/sys/time.h>
+#include <Kernel/API/POSIX/sys/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct rusage {
+    struct timeval ru_utime {
+    };
+    struct timeval ru_stime {
+    };
+    long ru_maxrss {};
```

Comment:
This isn't valid C, so if a port tries to include this from LibC it won't compile?

Reply:
I wondered for a minute but thought it was ok as it compiles